fusedwind/turbine/test/data/DTU10MW.dp3d

```diff
@@ -1,4 +1,5 @@
 # web00  web01  web02  web03
+# top  mid  mid  mid
```

Comment:
could we put these values on line 1 so we maintain backward compatibility? we can then just check whether the number of words is greater than 4 and then read these new keywords.

we should perhaps also add a version string as the first line to all the structure files, so any new features that will break backward compatibility can be dealt with. This could as well be done so files that don't have the version header can still be read.